### PR TITLE
Add EMR 6.11.0 and 6.12.0 Support

### PR DESCRIPTION
**Pull request by gapuchi**
EMR 6.11.0 and 6.12.0 have been released. 6.12.0 has some file changes that needs to be reflected in the validation. Added both releases and update the files to checked as part of this change.

*Issue #, if available:* https://github.com/awslabs/amazon-emr-serverless-image-cli/issues/2

*Description of changes:* Adding support for EMR 6.11.0 and 6.12.0 images.

Testing

```
python -m amazon_emr_serverless_image_cli validate-image -i public.ecr.aws/emr-serverless/spark/emr-6.9.0:latest -r emr-6.9.0 -t spark
python -m amazon_emr_serverless_image_cli validate-image -i public.ecr.aws/emr-serverless/hive/emr-6.9.0:latest -r emr-6.9.0 -t hive
python -m amazon_emr_serverless_image_cli validate-image -i public.ecr.aws/emr-serverless/spark/emr-6.10.0:latest -r emr-6.10.0 -t spark
python -m amazon_emr_serverless_image_cli validate-image -i public.ecr.aws/emr-serverless/hive/emr-6.10.0:latest -r emr-6.10.0 -t hive
python -m amazon_emr_serverless_image_cli validate-image -i public.ecr.aws/emr-serverless/spark/emr-6.11.0:latest -r emr-6.11.0 -t spark
python -m amazon_emr_serverless_image_cli validate-image -i public.ecr.aws/emr-serverless/hive/emr-6.11.0:latest -r emr-6.11.0 -t hive
python -m amazon_emr_serverless_image_cli validate-image -i public.ecr.aws/emr-serverless/spark/emr-6.12.0:latest -r emr-6.12.0 -t spark
python -m amazon_emr_serverless_image_cli validate-image -i public.ecr.aws/emr-serverless/hive/emr-6.12.0:latest -r emr-6.12.0 -t hive
```

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: amazon_emr_serverless_image_cli/assets/image-manifest.yaml [hunk 1]
```diff
@@ -1,5 +1,89 @@
...
+  Images:
+  - !ImageDetail
+    ImageType: spark
+    EnvironmentVariable:
+    - java_home
+    - spark_home
+    FileStructure:
+    - bin-files
+    - hadoop-files
+    - hadoop-jars-6.10
```
Comment (nithu0115) [inline amazon_emr_serverless_image_cli/assets/image-manifest.yaml:56]: should this hadoop jar version here be 6.11?

File: amazon_emr_serverless_image_cli/assets/image-manifest.yaml [hunk 2]
```diff
@@ -1,5 +1,89 @@
...
```
Comment (nithu0115) [inline amazon_emr_serverless_image_cli/assets/image-manifest.yaml:14]: should this hadoop jar version here be 6.12?

File: amazon_emr_serverless_image_cli/assets/image-manifest.yaml [hunk 3]
```diff
@@ -1,5 +1,89 @@
...
+      - hadoop_libexec_dir
+      - hadoop_user_home
+      - hadoop_yarn_home
+      - hive_home
+      - java_home
+      - tez_home
+      - yarn_home
+    FileStructure:
+      - hadoop-files
+      - hadoop-jars-6.10
```
Comment (nithu0115) [inline amazon_emr_serverless_image_cli/assets/image-manifest.yaml:35]: should this be 6.12?

File: amazon_emr_serverless_image_cli/assets/image-manifest.yaml [hunk 4]
```diff
@@ -1,5 +1,89 @@
...
```
Comment (nithu0115) [inline amazon_emr_serverless_image_cli/assets/image-manifest.yaml:77]: ditto

**Review by nithu0115 — CHANGES_REQUESTED** [review]

File: amazon_emr_serverless_image_cli/assets/image-manifest.yaml (hunk 1 above)
Comment (gapuchi) [reply amazon_emr_serverless_image_cli/assets/image-manifest.yaml:56]: 6.11 and 6.12 has the same files as 6.10, so using the same config. It's called 6.10 to distinguish from 6.9, which had different files.

**Review by vermanish — COMMENTED** [review]
Looks good

**Pull request merged**